<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simucontrol</class>
 <widget class="QWidget" name="Simucontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仿真控制</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;宋体&quot;;
background-color: rgb(231, 240, 245);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>200</width>
     <height>550</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>550</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;汉仪中黑简&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
   </property>
   <property name="title">
    <string>速度设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(224, 234, 245, 0), stop:1 rgba(255, 255, 255, 0));
    border-width:0px;</string>
    </property>
    <property name="text">
     <string>当前速度</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_simuctr_horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>39</y>
      <width>131</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0284091 rgba(221, 221, 221, 255), stop:0.511364 rgba(196, 196, 196, 255), stop:1 rgba(255, 255, 255, 255));
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="m_simuctr_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>502</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;新宋体&quot;;
color: rgb(4, 30, 103);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 0), stop:0.5 rgba(224, 234, 245, 0), stop:1 rgba(255, 255, 255, 0));
    border-width:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>200</width>
     <height>550</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>550</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;汉仪中黑简&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.528364, y1:1, x2:0.511, y2:0.0113636, stop:0 rgba(255, 255, 255, 100), stop:0.5 rgba(224, 234, 245, 100), stop:1 rgba(255, 255, 255, 100));
 border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    padding: 2px;</string>
   </property>
   <property name="title">
    <string>仿真控制</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>81</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonStart">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>仿真开始</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>82</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="buttonpause">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>仿真暂停</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>81</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="buttonStop">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>51</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>仿真停止</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>81</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>318</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_APPLY">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="m_simuctr_buttonEnter">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="m_simuctr_buttonCancel">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.520599, y1:1, x2:0.508, y2:0.142, stop:0 rgba(255, 255, 255, 255), stop:0.0564972 rgba(236, 241, 245, 255), stop:0.536723 rgba(236, 241, 245, 255), stop:1 rgba(255, 255, 255, 255));

    border-width: 1px;
    border-color: rgb(150, 194, 219);
    border-style: solid;
    border-radius:2;
    padding: 2px;
 
font: 13pt &quot;汉仪中黑简&quot;;
color: rgb(7, 100, 143);</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
